src/main/java/com/google/cloud/tools/appengine/cloudsdk/AppCfgRunner.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.cloudsdk;
+
+import com.google.cloud.tools.appengine.cloudsdk.internal.process.ProcessBuilderFactory;
+import com.google.cloud.tools.appengine.cloudsdk.process.ProcessHandler;
+import com.google.cloud.tools.appengine.cloudsdk.process.ProcessHandlerException;
+import com.google.common.base.Joiner;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+class AppCfgRunner {
```

Comment:
I'm not totally clear on why we're still calling appcfg. I thought we wanted to use gcloud for staging instead?

Reply:
Historically we've always called appcfg to do staging. Gcloud doesn't actually provide a mechanism for staging standalone. What it does is internally if deploying an exploded-war appengine standard project, it will call "stage" as part of deployment.